modules/ossm-deploy-application-workloads-per-user-group.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+// install/ossm-using-multiple-control-planes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-per-user-group_{context}"]
+= Deploy application workloads per user group
```

Comment:
```suggestion
= Deploy application workloads in each mesh
```
The term "user group" seems very specific to the example (taken from the [upstream Istio doc](https://istio.io/latest/docs/setup/install/multiple-controlplanes/)), but it is just an example and not an actual Istio concept, so I think saying "Deploy application workloads in each mesh" better describes the scenario.

Reply:
@longmuir Why even use the term "user group"? This document describes deploying two meshes on the same cluster, so why introduce an additional concept (and labels) when the point is simply to mark the mesh each namespace belongs to?

Reply:
Agreed. It was from the upstream Istio doc, and I removed a lot of its use here because it felt confusing, but I left it for the discoverySelector labels. I see you just used `mesh=mesh1` in your example doc for discovery selectors. I think you're right that it's good remove the term completely to avoid confusion that it is an Istio concept, rather than just a choice of naming.

I also like how you broke out the discovery selector labelling step up from the injection labelling, and I think that's a good idea too. 

I'll suggest changes for both - thanks!